spec/std/math/stats_spec.cr

```diff
@@ -0,0 +1,8 @@
+require "spec"
+
+describe "Stats" do
+  it "mean" do
```

Comment:
This could be either more descriptive and read as sentence. So, something like "calculates the mean". What do you say?